### PR TITLE
Don't break on ops parse errors

### PR DESCRIPTION
**Pull request by toddmedema**
Also looks like the regex processing perf improvement's in here, woops

### CODE REVIEW

File: lib/render/QDLParser.tsx [hunk 1]
```diff
@@ -6,8 +6,10 @@ import {LogMessage, LogMessageMap, Logger} from './Logger'
 import REGEX from '../Regex'
 
 export function renderXML(md: string): QDLParser {
+  const startTime = Date.now();
   const qdl = new QDLParser(XMLRenderer);
   qdl.render(new BlockList(md));
+  console.log('XML rendered in ' + (Date.now() - startTime) + 'ms');
```
Comment (smartin015) [inline lib/render/QDLParser.tsx]: Thoughts on rendering this as an info message instead of a console log?

File: lib/render/render/Renderer.tsx [hunk 2]
```diff
@@ -23,6 +23,16 @@ export interface Renderer {
 // ~~ to <del>
 export function sanitizeStyles(string: string): string {
 
+  // First, store and remove the contents of {{ops}} so they don't interfere with styling
+  const ops: string[] = [];
+  const opsRegex = /({{.*?}})/g; // non-greedily capture {{ops}}
```
Comment (smartin015) [inline lib/render/render/Renderer.tsx]: `{{counterexample = "}}" }}`

File: lib/render/QDLParser.tsx (hunk 1 above)
Comment (toddmedema) [reply lib/render/QDLParser.tsx]: Sure, suppose that's a bit more semantically correct. Done.

File: lib/render/render/Renderer.tsx (hunk 2 above)
Comment (toddmedema) [reply lib/render/render/Renderer.tsx]: Good catch, fixed + added (passing) test case

File: lib/render/QDLParser.tsx (hunk 1 above)
Comment (smartin015) [reply lib/render/QDLParser.tsx]: Ah, I meant with https://github.com/ExpeditionRPG/expedition-qdl/blob/master/lib/render/Logger.tsx#L64

You could move the timing details into the call to qdl.render and then use `this.log.dbg`. The debug details are then grouped with the other details.

File: lib/render/render/Renderer.tsx [hunk 3]
```diff
@@ -25,7 +25,7 @@ export function sanitizeStyles(string: string): string {
 
   // First, store and remove the contents of {{ops}} so they don't interfere with styling
   const ops: string[] = [];
-  const opsRegex = /({{.*?}})/g; // non-greedily capture {{ops}}
+  const opsRegex = /({{.*?[^"}]}})/g; // non-greedily capture {{ops="}}"}}
```
Comment (smartin015) [inline lib/render/render/Renderer.tsx]: `{{ secondCounterexample = {"a": {"b": 5}} && false }}`

We could try tweaking something like [https://regex101.com/r/eE6mX3/1] (which I got from https://stackoverflow.com/questions/546433/regular-expression-to-match-outer-brackets)

**Pull request merged**